data_steward/retraction/retract_data_bq.py

```diff
@@ -359,65 +233,100 @@ def get_retraction_queries_fact_relationship(client: BigQueryClient,
     :return: list of queries
     """
     if not client.table_exists(FACT_RELATIONSHIP, dataset_id):
-        LOGGER.info(f"{FACT_RELATIONSHIP} does not exist.")
+        LOGGER.info(f"Skipping {FACT_RELATIONSHIP}. It does not exist.")
```

Comment:
You could set your retraction action list here, right?
```python
action_list = ['delete'] if skip_sandboxing else ['sandbox', 'delete']
...
for action in action_list:
```
This might be easier to read and maintain than packing this into the same loop multiple times (which could lead to multi-line updates later??).